umeå/meetup.yaml

```diff
@@ -0,0 +1,86 @@
+city: Umeå
+country: sweden
+meetupID: Cloud-Native-Northern-Sweden
+meetups:
+- address: Tvistevägen 47
+  attendees: 19
+  date: "2019-05-28T17:00:00Z"
+  duration: 2h0m0s
+  id: 261450080
+  name: 'Trends in cloud native - a summary from KubeCon + CloudNativeCon Barcelona '
+  presentations:
+  - endTime: "18:30"
+    slides: ""
+    speakers:
+    - lindaalmersson
+    - tordsson
+    startTime: "17:00"
+    title: ""
+  sponsors:
+    other: null
+    venue: elastisys
+- address: Västra Norrlandsgatan 10D
+  attendees: 36
+  date: "2019-05-07T17:00:00Z"
+  duration: 2h0m0s
+  id: 260276456
+  name: What does “production ready” really mean for a Kubernetes cluster?
+  presentations:
+  - endTime: "18:30"
```

Comment:
I fixed endTime/startTime at latest master, now just put `delay: 30m0s` and `duration: 60m0s` instead (for all presentations)